api/server/logs.go

```diff
@@ -0,0 +1,42 @@
+package server
```

Comment:
Why does this need to access to anything in `server`? Is it because of `ES`? That can be passed as a parameter, right?